AppInspector/AppInspector.Commands.csproj

```
@@ -84,9 +82,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\LICENSE.txt" Pack="true" PackagePath=""/>
-    <None Include="..\icon-128.png" Pack="true" PackagePath=""/>
-    <None Update="rules\**" Pack="true" PackagePath="lib\$(TargetFramework)\"/>
+    <None Include="..\LICENSE.txt" Pack="true" PackagePath="" />
+    <None Include="..\icon-128.png" Pack="true" PackagePath="" />
   </ItemGroup>
 
+  <Target Name="PreBuild" BeforeTargets="BeforeCompile">
+    <Exec Command="..\RulesPacker\ApplicationInspector.CLI.exe packrules -r rules\default -o Resources\defaultRules.json" />
```

use `dotnet ../RulesPacker/ApplicationInspector.CLI.dll` as it will be cross platform

---

This from DevSkim works on Windows and Mac

```
<Target Name="PreBuild" BeforeTargets="BeforeCompile">
    <Exec Command="dotnet ../../RulePacker/devskim.dll pack ../../rules Resources/devskim-rules.json" />
  </Target>
```
```suggestion
    <Exec Command="dotnet ../RulesPacker/ApplicationInspector.CLI.exe packrules -r rules/default -o Resources/defaultRules.json" />
```

---

This could be what is breaking the linux/mac build.

---

it complains it can't find hostpolicy.dll when I use the dll method but yes, makes sense for other OS's.  I'll review to address.

---

also using the / stead of \ causes the build to fail as it can't locate the file i.e. error 9009 when building on windows.  At this point I welcome some help to short cut the final commit to address if you're interested.

---

I'm checking out the code now.

---

Ok.  I did get it working locally complete with \ and dll but I see you committed a change that is similar.  I had also removed the unnecessary files including the defaultRules.json.  Thanks!  We'll go with the changes you made.